docs/en/admins/16_Oidc.md

```diff
@@ -0,0 +1,27 @@
+# OIDC
```

Comment:
I prefer to have the full name instead of just the abbreviation. 
The same for the file name.

I would suggest: `16_OpenID-Connect.md` and `# OpenID Connect (OIDC)` as headline